### PR TITLE
Coding Task Inci

### PR DESCRIPTION
**Pull request by inciigin**

### CODE REVIEW

File: src/Week7_CodingTasks/Task1_Inci.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+                min = arr[i];
+            }
+        }
+        return min;
+    }
+
+    public static void main(String[] args) {
+        int[] arr = {1, 2, 3, 4, 5, 7, 9, 3,-1};
+        int minNumber = findMinimumNumber(arr);
+        System.out.println("The maximum number in the array is: " + minNumber);
```
Comment (Kallen04) [inline src/Week7_CodingTasks/Task1_Inci.java]: The code is good, just need to change the wording here from maximum to minimum

File: src/Week7_CodingTasks/Task1_Inci.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package Week7_CodingTasks;
+
+public class Task1_Inci {
+
+    /*
+    Question1: Array - Find Minimum
+Write a method that can find the maximum number from an int Array
```
Comment (Kallen04) [inline src/Week7_CodingTasks/Task1_Inci.java]: And here too, need to be minimum, not maximum

**Review by Kallen04 — APPROVED** [review]
Thumbs up!

File: src/Week7_CodingTasks/Task1_Inci.java (hunk 1 above)
Comment (inciigin) [reply src/Week7_CodingTasks/Task1_Inci.java]: Ops , Got it :|

**Pull request merged**